### DISCUSSION TITLE
Amazon Fire Tablet not working with Qdomyos?

### DISCUSSION THREAD

**Discussion by pbcsd** [cagnulein/qdomyos-zwift#651] [Q&A] [answered]
So the tablet seems to connect fine,recognizes when I first start pedaling but then none of the metrics change? The app isn't frozen as the clock continues to work but that's the only thing changing?

![20220206_081203](https://user-images.githubusercontent.com/99127458/152682927-32534de0-a6a6-4a92-bd3d-a1b8b6fff625.jpg)

Eg it stays as 13.1 and the 25 seconds doesn't change?

**Comment by cagnulein** [comment, marked as answer]
hi, try to disable the virtual device setting in the experimental setting in QZ and restart it @pbcsd

**Reply by pbcsd** [reply]
Perfect... Thank you!
